tile.yml

```diff
@@ -214,6 +214,20 @@ forms:
     description: A comma delimited list of roles (minus the role/ prefix) that can
       be used when creating bound users for this service
     configurable: true
+- name: compatibility
+  label: Compatibility
+  description: Legacy Compatibility Options
+  properties:
+  - name: gsb_compatibility_three_to_four
+    type: boolean
+    default: false
+    label: Compatibility with GCP Service Broker v3.X
+    description: Enable compatibility with the GCP Service Broker v3.x. Before version
```

Comment:
This is a stretch, but would this be a good place for the `>` block scalar style?

Reply:
This is related to the bug I found in the tile generator, it seems like their code only supports basic strings so I had to write a function to normalize our description: https://github.com/cf-platform-eng/tile-generator/issues/280